### PR TITLE
A11y badge

### PR DESCRIPTION
**Pull request by spacerockzero**

### CODE REVIEW

File: badges-active/accessibility/_micro_semantic-html.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Accessibility - Semantic HTML [Micro Badge]
```
Comment (spacerockzero) [inline badges-active/accessibility/_micro_semantic-html.md]: Should this replace the HTML - semantic html micro? I like yours better, and we can reference the same micro from each category.

File: badges-active/accessibility/journeyman.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+### Earn all of these:
+
+- [Screen Readers (~3h)](_micro_screen-readers.md)
+- [ARIA (~2h)](_micro_aria.md)
+- [Focus Management (~1h)](_micro_focus-management.md)
+- [Testing and Auditing Tools (~2-3h)](_micro_testing-and-auditing-tools.md)
+
+### Once above are met, earn 1 of these:
+
+- [AccessibilitySS Explorer (~1h)](_micro_EXPLORER.md)
```
Comment (spacerockzero) [inline badges-active/accessibility/journeyman.md]: Not sure what the extra letters in that title are there for. Isn't it just "Accessibility Explorer"?

**Comment by spacerockzero** [comment]
Great work. Merged it.

File: badges-active/html/apprentice.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Estimated time: ~4.5h
 
 **Elements**
 
-- [ ] [Semantic HTML5 Elements (~0.5h)](_micro_semantic-html5-elements.md)
```
Comment (rosemariesadler) [inline badges-active/html/apprentice.md]: did we really intend to delete the semantic elements html5 micro badge?

Comment (straker) [reply badges-active/html/apprentice.md]: Yes, it was replaced by the a11y semantic html micro, which now counts for both badges.

**Pull request merged**